sites/kit.svelte.dev/src/routes/home/Deployment.svelte

```diff
@@ -0,0 +1,243 @@
+<script>
+	import Section from '@sveltejs/site-kit/components/Section.svelte';
+	import html5 from './logos/html5.svg';
+	import node from './logos/node.svg';
+	import vercel from './logos/vercel.svg';
+	import netlify from './logos/netlify.svg';
+	import cloudflare from './logos/cloudflare.svg';
+	import firebase from './logos/firebase.svg';
+	import deno from './logos/deno.svg';
+	import lambda from './logos/lambda.svg';
+	import appengine from './logos/appengine.svg';
+	import plus from '$lib/icons/plus.svg';
+</script>
+
+<Section --background="var(--background-1)">
+	<div class="grid" style="--columns: 3">
+		<h2>deploy anywhere</h2>
+		<div class="blurb">
+			<p>
+				Export static HTML files. Run your own Node server. Deploy code to the edge of the world. If
+				a platform runs JavaScript, it runs SvelteKit — in some cases with <strong
+					>zero configuration</strong
+				>.
+			</p>
+			<p>Want to try deploying somewhere else? Swap out your adapter with a single line of code.</p>
+		</div>
+	</div>
+
+	<div class="grid" style="--columns: 3">
+		<div class="platforms left">
+			<div class="invert">
+				<img src={html5} alt="HTML5 logo" />
+				<span><span class="large">Static</span> HTML</span>
+			</div>
+			<div class="invert">
+				<img src={node} alt="Node logo" />
+				<span>Node.js</span>
+			</div>
+			<div class="invert invert-hover">
+				<img src={vercel} alt="Vercel logo" style="transform: translate(0,-0.2rem)" />
+				<span>Vercel</span>
+			</div>
+			<div>
+				<img src={netlify} alt="Netlify logo" />
+				<span>Netlify</span>
+			</div>
+			<div>
+				<img src={cloudflare} alt="Cloudflare logo" />
+				<span>Cloudflare</span>
+			</div>
+		</div>
+
+		<div class="platforms right">
+			<div>
+				<img src={firebase} alt="Firebase logo" />
+				<span>Firebase</span>
+			</div>
+			<div class="invert invert-hover">
+				<img src={deno} alt="Deno logo" />
+				<span>Deno</span>
+			</div>
+			<div>
+				<img src={lambda} alt="AWS Lambda logo" />
+				<span>AWS</span>
+			</div>
+			<div>
+				<img src={appengine} alt="AppEngine logo" />
+				<span>AppEngine</span>
+			</div>
+			<div>
+				<img src={plus} alt="Plus sign" />
+				<span>More...</span>
+			</div>
+		</div>
+
+		<div class="globe">
```

Comment:
The background image is not being exposed to screen readers. We may need to [do something like this](https://www.davidmacd.com/blog/alternate-text-for-css-background-images.html) to expose alt text (assuming we can't use an actual `<img>`).

Reply:
Yeah, we have to use a background image so that we can vary the source based on `prefers-color-scheme`. Fixed